content/en/contribute/code/workflow.md

```diff
@@ -96,7 +96,7 @@ Never force push remote. Prefer rebasing over merging as it makes for a cleaner
 
 Commit reformats and refactors separately from actual code changes to make reviewing easier.
 
-For more help with Git see: [Using Git]({{% ref "core/guides/using-git" %}}).
+For more help with Git see: [Using Git](https://lab.github.com/).
```

Comment:
The url used is mostly about the GitHub product/tool but the context of this line is more about the git cool. Suggestion here is to point to the official git site where they provide ways to learn

```suggestion
For more help with Git see: [Using Git](https://git-scm.com/doc/ext).
```

Reply:
yeah - wanted feature parity w/ the other link - but yeah `git` != GH